.changeset/tender-suits-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/markdown-remark': patch
+---
+
+Adds experimental support for multiple themes in `shikiji`.
```

Comment:
```suggestion
---
'@astrojs/markdown-remark': minor
'astro': minor
---

Adds experimental support for multiple shiki themes with the new `markdown.shikiConfig.experimentalThemes` option.
```

Extended the changeset a bit to highlight the new schema option, and marked minor as these will be new features. I think we can merge this on next Wednesday when we usually do minors.

Reply:
Love this change, @bluwy !